src/app/modules/shared_models/message_item.nim

```diff
@@ -72,7 +72,7 @@ proc initItem*(
   result.senderIcon = senderIcon
   result.seen = seen
   result.outgoingStatus = outgoingStatus
-  result.messageText = text
+  result.messageText = if ContentType.Image == contentType: "" else: text
```

Comment:
How about moving the placeholder text from `chat.nim` to here if the `text` role is empty?

Reply:
@stefandunca image should be sent without text, that not empty `text` property is required by status-go protocol.